dask/array/routines.py

```diff
@@ -1405,7 +1405,17 @@ def piecewise(x, condlist, funclist, *args, **kw):
 
 
 def aligned_coarsen_chunks(chunks: List[int], multiple: int) -> List[int]:
-    """ Returns a new chunking aligned with the coarsening multiple"""
+    """
+    Returns a new chunking aligned with the coarsening multiple.
+    Any excess is at the end of the array.
+
+    Examples
+    --------
+
+    chunks=(1, 2, 3); multiple=4; returns (4, 4, 2)
+    chunks=(1, 20, 3, 4); multiple=4; returns (20, 4, 4)
+    chunks=(20, 10, 15, 23, 24), multple=10; returns (20, 10, 20, 20, 20, 2)
+    """
```

Comment:
I recommend instead turning these into doctests

```suggestion
    >>> align_coarsen_chunks(chunks=(1, 2, 3), multiple=4)
    (4, 4, 2)
    >>> align_coarsen_chunks(chunks=(1, 20, 3, 4), multiple=4)
    (20, 4, 4)
    >>> align_coarsen_chunks(chunks=(20, 10, 15, 23, 24), multple=10)
    (20, 10, 20, 20, 20, 2)
    """
```

This way they get tested, and follow a familiar pattern in the codebase.

Reply:
Ah got it. :+1: